ribs/emitters/_iso_line_emitter.py

```diff
@@ -151,15 +158,9 @@ def ask(self):
 
         if self.archive.empty:
             solution_batch = np.expand_dims(self._x0, axis=0) + iso_gaussian
```

Comment:
iso_gaussian should only be computed in this if statement since it's not used anywhere else